### DISCUSSION TITLE
Smoke control area

### DISCUSSION THREAD

**Discussion by kieranwint** [digital-land/data-standards-backlog#86] [Planning consideration]
A smoke control area (defined by [UK Air (DEFRA)](https://uk-air.defra.gov.uk/sca/)) is an area where people and businesses must not:

- Emit smoke from a chimney
- Buy or sell unauthorised fuel for use in a smoke control area unless it’s used in an ‘exempt’ appliance (appliances which are approved to use in smoke control areas)

We are looking at smoke control areas as it is one of the [65 planning considerations or datasets The Geospatial Commission has identified as being used during the housing and planning process](https://www.gov.uk/government/publications/bringing-together-housing-land-and-planning-geospatial-datasets).

During the initial stages of research, we found that the data had an unclear licence, and so we have blocked this for now while we dig into this a little deeper.

We are still very interested to hear any thoughts on this dataset and the value it may provide.

**Comment by drewhardy77** [comment]
New data endpoint confirms data is available under OGL3 https://environment.data.gov.uk/dataset/b21eb709-15e3-4ae7-8d3c-132903b7ba4f
